app/cdap/components/WranglerV2/SearchResultsListItem/SearchResultListItem.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+import SearchResultListItem from 'components/WranglerV2/SearchResultsListItem';
+
+export default {
+  title: 'SearchResultListItem',
+  component: SearchResultListItem,
+};
```

Comment:
follow typescript example

Reply:
updated the story with typescript template